packages/token/src/common/erc2981/interface.cairo

```diff
@@ -5,10 +5,72 @@ use starknet::ContractAddress;
 
 pub const IERC2981_ID: felt252 = 0x2d3414e45a8700c29f119a54b9f11dca0e29e06ddcb214018fc37340e165ed6;
 
+/// Interface of the ERC2981 standard.
 #[starknet::interface]
 pub trait IERC2981<TState> {
     /// Returns how much royalty is owed and to whom, based on a sale price that may be denominated
     /// in any unit of exchange. The royalty amount is denominated and should be paid in that same
     /// unit of exchange.
     fn royalty_info(self: @TState, token_id: u256, sale_price: u256) -> (ContractAddress, u256);
 }
+
+/// Interface providing external read functions for discovering the state of ERC2981 component.
+#[starknet::interface]
+pub trait IERC2981StateInfo<TState> {
```

Comment:
What do you think about calling this `IERC2981Info` to shorten the name?

Reply:
We can totally do that